site/src/components/GlobalSnackbar/EnterpriseSnackbar.tsx

```diff
@@ -35,7 +35,19 @@ export const EnterpriseSnackbar: FC<EnterpriseSnackbarProps> = ({
   action,
   ...snackbarProps
 }) => {
-  const content = useClassName(classNames.content(variant), [variant]);
+  const content = useClassName(
+    (css, theme) => css`
+      border: 1px solid ${theme.palette.divider};
+      border-left: 4px solid ${variantColor(variant, theme)};
+      border-radius: 8px;
+      padding: 8px 24px 8px 16px;
+      box-shadow: ${theme.shadows[6]};
+      align-items: inherit;
+      background-color: ${theme.palette.background.paper};
+      color: ${theme.palette.text.secondary};
+    `,
+    [variant],
+  );
```

Comment:
I'm not convinced the lint rule is worth needing to inline large CSS definitions like this. feels really noisy, and I've been actively working to move stuff like this out of the components. 😅

Reply:
Yeah, when I was making the changes, it felt like it was a trade-off between clean code and fragile code (making sure the component doesn't render with stale values)

But now that you're pushing back on this, I think you're right. I'm going to see if there might be a better way to thread the needle and get both. It'll have to wait until the weekend, though – I'm working on the tests outside of Backstage hours